### PR TITLE
chore: Add Edit/Delete replies button to Circles

### PR DESCRIPTION
**Pull request by bruce-glazier**
## Changes

  - Your changes...

## Screenshots

https://github.com/lifeomic/react-native-sdk/assets/76954025/409d508a-52f1-4628-bca2-bddc15f76e85

### CODE REVIEW

File: lib/i18n/index.ts [hunk 1]
```diff
@@ -38,15 +38,6 @@ export function configure() {
     .init({
       fallbackLng: 'en',
       supportedLngs: ['en', 'es', 'fr', 'ar', 'tr', 'pt', 'de'],
-      resources: {
```
Comment (bruce-glazier) [inline lib/i18n/index.ts:41]: Not sure why but this seemed like it was needed for tests but now it just overwrites the resources loaded from the resource loader plugin.

File: src/components/Circles/ThreadPost.tsx
Comment (bruce-glazier) [inline src/components/Circles/ThreadPost.tsx:1]: This was basically identical to PostItem `formerly Post` so I figured we should just consolidate these into one.

File: src/components/Circles/PostsList.tsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ export const PostsList = ({ circleTile, onOpenPost }: PostsListProps) => {
     useInfinitePosts({
       circleId: circleTile?.circleId,
     });
+  // useFocusEffect(() => {});
```
Comment (sternetj) [inline src/components/Circles/PostsList.tsx:32]: Is this needed?

**Review by sternetj — APPROVED** [review]

File: src/components/Circles/PostsList.tsx (hunk 2 above)
Comment (bruce-glazier) [reply src/components/Circles/PostsList.tsx:32]: I have a fast follow that removes this almost ready for review.

**Pull request merged**